features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/AsyncApiConsole/AsyncApiUI.jsx

```diff
@@ -111,23 +152,42 @@ export default function AsyncApiUI(props) {
     } else {
         return (
             <>
-                {api.type === 'WEBSUB' && allTopics.list.map((topic, index) => (
+                <FormControl className={classes.endpointSelectorRoot}>
+                    <InputLabel>Servers</InputLabel>
+                    <Select
+                        className={classes.selectList}
+                        id="api-endpoint-select"
+                        value={endPoint}
+                        displayEmpty
+                        onChange={handleServerChange}
+                    >
+                        {Object.entries(URLs).map(([key, value]) => {
+                            if (value) {
+                                return <MenuItem value={value}>{value}</MenuItem>;
```

Comment:
MenueItem should have a key when population list of elements from an array.

```
A “key” is a special string attribute you need to include when creating lists of elements
```
https://reactjs.org/docs/lists-and-keys.html#keys